css/estilos-movies.css

```diff
@@ -0,0 +1,152 @@
+@import url(flexboxgrid.min.css);
+
+.seccion-dos{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: center;
+    /*background-color: #38bb6a;*/
+    margin-top: 15px;
+    margin-bottom: 25px;
+    margin-right: 5px;
+    margin-left: 5px;
+    align-items: flex-start;
+}
+.seccion1movies{
+    float: left;
+    width: 100%;
+    color: #370617;
+    margin: 10px 0 0 45px;
+    padding: 0;
+    font-size: 20px;
+    text-align: center;
+    font-family: 'Mulish', sans-serif;
+}.listapeliculas{
+    display: flex;
+    justify-content: end;
+}
+.listapeliculas{
+    background-color: #fff;
+    color: #370617;
+    border: 1px solid #370617;
+    letter-spacing: 2px;
+    text-transform: unset;
+    cursor: pointer;
+    padding: 15px;
+    width: 95%;
+    margin: 5px 25px 0 0;
+    font-size: 18px;
+    text-align: center;
+    text-decoration: none;
+    font-family: 'Mulish', sans-serif;
+}
+
+.listapeliculas:hover{
+    background-color: #012a4a;
+    color: #fff;
+    text-decoration: underline;  
+}
+
+.seccion2movies{
+    width: 30%;
+    float: right;
+    margin: 0 0 0 0;
+}
+.lista{
+    width: 20%;
+    float:right;
+}
+#seccion3movies{
+    width: 100%;
+    justify-items: center;
+}
+.seccion3{
+    width: 30%;
+    color: #370617;
+    padding: 10px;
+    text-align: center;
+    float: left;

```

Comment:
Floating this element will cause overflow outside its container. Another way to achieve this would be to use `display: inline-block` instead.